src/sidebar/test/annotation-fixtures.js

```diff
@@ -113,7 +113,7 @@ function oldHighlight() {
   return {
     id: 'annotation_id',
     $highlight: undefined,
-    target: ['foo', 'bar'],
+    target: [{ source: 'source', selector: [] }],
```

Comment:
In other `annotation-metadata` logic, annotations with no `selector` on their `target[0]` element are considered `pageNotes`, which makes them de-facto not highlights.